spec/unit/mutant/integration/rspec_spec.rb

```diff
@@ -1,10 +1,10 @@
 require 'mutant/integration/rspec'
 
 RSpec.describe Mutant::Integration::Rspec do
-  let(:object) { described_class.new }
+  let(:object) { described_class.new(Mutant::Config::DEFAULT) }
 
-  let(:options)       { double('options')                           }
-  let(:runner)        { double('runner')                            }
+  let(:options) { double('options') }
```

Comment:
I want to introduce `instance_double` in a global refactoring. Right now mutant consistently uses the weaker `double` form. To keep this consistency lets continue with `double` for this PR.